time[5] time_stamp
int32[5] point_num
string[5] point_name
int32[5] point_score
